lib/gn.py

```diff
@@ -112,6 +113,7 @@ def download(self, ud, d):
 
         uri = ud.url.split(";")[0]
 
+        ud.trying_to_fetch_with_gclient = True
```

Comment:
Shouldn't this be set to False after a successful fetch?

Reply:
Yeah, it's a good idea to put it back to False.